### DISCUSSION TITLE
Can I Embed audio into a field or do I need a custom field to do so

### DISCUSSION THREAD

**Discussion by Gian-HOF** [firecmsco/firecms#443] [Q&A]
I want to create a blog and want to host all my content on fireCMS, my blog page will have three things on it, an audio player, like a podcast, and underneath that a transcription of the audio, and then a button somewhere to download a PDF version of the transcription.

I assume downloading a file is easy enough and having text on the page is kinda the whole point of a blog, but I'm not sure if I'll be able to have audio embedded or stored on FireCMS and when it gets pulled into the web app or android/IOS app it displays as an audio player.

I'll be grateful for any info or help

**Comment by fgatti675** [comment]
Sure, if you use a regular storage field (a string property with the storage config set), files that are detected as audio will be previewed with the default player of the browser:
```
buildProperty({
    dataType: "string",
    name: "Image",
    storage: {
        storagePath: "audios",
        acceptedFiles: ["audio/*"]
    }
});
```
The `acceptedFiles` is only used for filtering the files you are allowed to upload
